app/views/devise/shared/_registration_form.html.slim

```diff
@@ -23,6 +23,8 @@
     .form-control-position
       i.fa.fa-envelope
     .help-block.font-small-3
+  = f.hidden_field :role, :value => User::ADMIN_ROLE
```

Comment:
move to controller. hidden fields for role is very bad practice. everyone could look in html body